Modsen.Server.Authentication.Application/Helpers/ConfigurationHelper.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.Configuration;
+using Modsen.Server.Authentication.Domain.Exceptions;
+
+namespace Modsen.Server.Authentication.Application.Helpers
+{
+    public static class ConfigurationHelper
+    {
+        public static int GetRefreshTokenValidityInDays(IConfiguration configuration)
+        {
+            var parseResult = int.TryParse(configuration["JWT:RefreshTokenValidityInDays"], out int refreshTokenValidityInDays);
+
+            if (parseResult)
+            {
+                return refreshTokenValidityInDays;
+            }
+
+            throw new BadRequestException();
```

Comment:
BadRequest is for requests coming from user. This is about configuration, if i understood right